src/components/expanded-state/SupportedCountriesExpandedState.js

```diff
@@ -1,39 +1,111 @@
-import React from 'react';
 import { values } from 'lodash';
-import { StyleSheet, Text } from 'react-native';
-import FloatingPanels from './FloatingPanels';
-import { AssetPanel } from './asset-panel';
+import React from 'react';
+import { StyleSheet } from 'react-native';
+import { useDimensions } from '../../hooks';
+import isNativeStackAvailable from '../../helpers/isNativeStackAvailable';
 import { supportedCountries } from '../../references/wyre/supportedCountries';
-import { Centered } from '../layout';
-import { padding } from '../../styles';
+import { colors, fonts, padding } from '../../styles';
 import { deviceUtils } from '../../utils';
+import { ButtonPressAnimation } from '../animations';
+import { FloatingEmojis, FloatingEmojisTapHandler } from '../floating-emojis';
+import { Centered } from '../layout';
+import { Text } from '../text';
+import { AssetPanel } from './asset-panel';
+import FloatingPanels from './FloatingPanels';
 
 const sx = StyleSheet.create({
-  text: {
-    fontSize: 13,
-    fontWeight: 'normal',
-    lineHeight: 20,
+  body: {
+    color: colors.alpha(colors.blueGreyDark, 0.6),
+    fontSize: parseFloat(fonts.size.smedium),
+    lineHeight: 22,
+    textAlign: 'center',
+  },
+
+  footer: {
+    fontSize: parseFloat(fonts.size.smedium),
+    marginTop: 12,
     textAlign: 'center',
   },
 
   title: {
+    fontSize: parseFloat(fonts.size.large),
     fontWeight: 'bold',
-    marginBottom: 20,
+    marginBottom: 10,
+    textAlign: 'center',
   },
 });
 
-const countries = values(supportedCountries).map(c => c.name);
-const countriesList = `${countries.join(', ')}.`;
+const countries = values(supportedCountries).map(c =>
+  c.name == 'United States'
+    ? 'United\xa0States (except CT, HI, NC, NH, NY, VA, VT)'
+    : c.name.replace(/ /g, '\xa0')
+);
+const countriesList = `${countries.join(', ')}`;
+
+const emojiArray = [
```

Comment:
We should reuse the list from here: https://github.com/rainbow-me/rainbow/blob/%40mike/final-add-cash/src/references/wyre/supportedCountries.js#L4

Something like 

```
const emojiArray = values(supportedCountries).map(c => c. emojiName);
```